app/presenters/schedule_presenter.rb

```diff
@@ -16,7 +16,8 @@ def self.record_for_microservices(schedule)
       monthOfYear: schedule.month_of_year,
       year: schedule.year,
       createdAt: schedule.created_at,
-      updatedAt: schedule.updated_at
+      updatedAt: schedule.updated_at,
+      tempr_url: schedule.tempr_url
```

Comment:
Why are we not camel casing this? The microservices should by default use camel case, (also handling snake in some cases for backwards compatibility).

Reply:
Because in devices it's snake case still.